src/Microsoft.TestPlatform.Build/Microsoft.TestPlatform.targets

```diff
@@ -52,7 +52,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     />
   </Target>
 
-  <Target Name="ShowInfoMessageIfProjectHasNoIsTestProjectProperty" Condition="'$(IsTestProject)' != 'true'">
+  <Target Name="ShowInfoMessageIfProjectHasNoIsTestProjectProperty" Condition="'$(IsTestProject)' == ''">
```

Comment:
If solution contains non-test dlls, then currently we are logging dll skip message as info. Ideally this message should be logged as verbose level.

If possible, can we make log level of skipping test dll message as verbose?

Reply:
There is no infra like that in place today.

Reply:
> There is no infra like that in place today.

I beg to differ. `VSTestLogsTask` inherits from `Task`, which provides a logger. `VSTestLogsTask` can use that logger, with appropriate importance levels, instead of `Console.WriteLine`. Here's a PR: https://github.com/Microsoft/vstest/pull/1882